configuration.nix

```diff
@@ -13,6 +14,9 @@ in
   # WSL is closer to a container than anything else
   boot.isContainer = true;
 
+  # Include Windows %PATH% in Linux $PATH.
+  environment.extraInit = ''PATH="$PATH:$WSLPATH"'';
```

Comment:
That has one downside: It slows down tab completion down tremendously because looking up the windows directories is very very very slow.

See https://github.com/akinomyoga/ble.sh/issues/96#issuecomment-912952486= for some details.

Reply:
From what I understand, this is a problem for all WSL distros. Appending the path can be toggled off with `wsl.wslConf.interop.appendWindowsPath = false;`. #64 has an option to toggle off the extraInit code

Reply:
Yes, it is sadly.